.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,98 @@
+name: Bug report
+description: File a bug report to improve Superset's stability
+labels: ["bug"]
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: How to reproduce the bug
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots-recordings
+    attributes:
+      label: Screenshots/recordings
+      description: If applicable, add screenshots or recordings to help explain your problem.
+  - type: markdown
+    attributes:
+      value: |
+        ### Environment
+
+        Please specify your environment. If your environment does not match the alternatives,
+        you need to upgrade your environment before submitting the issue as it may have already been fixed.
+        For additional information about the releases, check out the [release page](https://github.com/apache/superset/wiki).
+  - type: dropdown
+    id: superset-version
+    attributes:
+      label: Superset version
+      options:
+        - master
+        - "3.1.0"
+        - "3.0.3"
+    validations:
+      required: true
+  - type: dropdown
+    id: python-version
+    attributes:
+      label: Python version
+      options:
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - Not applicable
+        - I don't know
+    validations:
+      required: true
+  - type: dropdown
+    id: node-version
+    attributes:
+      label: Node version
+      options:
+        - "16"
+        - "17"
+        - "18 or greater"
+        - Not applicable
+        - I don't know
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser
+      options:
+        - Chrome
+        - Firefox
+        - Safari
+        - Not applicable
+    validations:
+      required: true
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional context
+      description: |
+        Add any other context about the problem here such as the feature flags that you have enabled,
+        any customizations you have made, etc.
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      description: Make sure to follow these steps before submitting your issue - thank you!
+      options:
+        - label: I have searched Superset docs and Slack and didn't find a solution to my problem.
+        - label: I have checked the issue tracker for the same issue and I haven't found one similar.
+        - label: I have checked Superset logs for Python stacktraces and included it here as text if there are any.
```

Comment:
```suggestion
        - label: I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.
```

Reply:
What section above do we want them to include it in?  I took a stab.  I wonder if there's a way for them to format it as code.  If the "additional context" section supports the three tick marks, we could include that in the instruction.

Reply:
The `Additional context` is the right section 👍🏼 

It's a `textarea` and has all the capabilities of a GitHub comment such as code formatting, attachments, etc.